### ISSUE TITLE
Tax Exempt permission applies to plots owned in other towns.

### ISSUE THREAD

**Issue by LlmDl** [LlmDl/Towny#2351]
Originally reported on Google Code with ID 2329

```
The Tax exempt permission could benefit fro the below listed feature.

Currently town staff etc do not pay tax anywhere. This currently applies as a blanket
rule.

Suppose the mayor of town A has embassy plots, then any mayor from any town can claim
those embassy plots and will never have to pay tax to Town A.

Could we rather have a different system where we have 

1) towny.tax_exempt.own where they dont pay tax for plots in their own town. But will
pay other towns if they have plots claimed from the other town.

2) towny.tax_exempt.all  where they will never pay tax of any sort.


Thank you for your fine work,
Regards,
Azfaloth

```

Reported by `Azfaloth` on 2015-07-30 07:08:05

**Comment by Azfaloth** [comment]
Hi, I'd like to bump this issue. :) Hopefully you will find some time to implement this.

Currently the way the plugin is, town mayors can own any number of plots in anyone elses town(embassy plots) and have zero tax. Towns do not want to set & sell embassy plots because other mayors can just buy them all up and never have to pay any tax for those plots ever.

Thanks a lot for your time and efforts,

Regards,
Azfaloth

**Comment by LlmDl** [comment]
Fixed in 0.91.4.8, mayors are no longer exempt from plottaxes and embassyplottax in towns they do not belong to.
